009-quarkus-infinispan-grpc-kafka/src/main/resources/application.properties

```diff
@@ -1,13 +1,18 @@
 
 # Infinispan
+quarkus.infinispan-client.trust-store=src/main/resources/server.jks
+quarkus.infinispan-client.trust-store-password=changeit
+quarkus.infinispan-client.trust-store-type=jks
+
 quarkus.infinispan-client.server-list=localhost:11222
 quarkus.infinispan-client.use-auth=true
-quarkus.infinispan-client.auth-username=my_username
-quarkus.infinispan-client.auth-password=my_password
-quarkus.infinispan-client.auth-realm=default
-quarkus.infinispan-client.auth-server-name=infinispan
-quarkus.infinispan-client.client-intelligence=BASIC
-quarkus.infinispan-client.sasl-mechanism=DIGEST-MD5
+# credentials is picked at build time from hotrod-client.properties
+#quarkus.infinispan-client.auth-username=my_username
+#quarkus.infinispan-client.auth-password=my_password
+#quarkus.infinispan-client.auth-realm=default
+#quarkus.infinispan-client.auth-server-name=infinispan
+#quarkus.infinispan-client.client-intelligence=BASIC
+#quarkus.infinispan-client.sasl-mechanism=DIGEST-MD5
```

Comment:
I like the comment, but IMO it's redundant to leave props here.
Comment can state that: Infinispan client credentials are picked at build time from hotrod-client.properties